### PR TITLE
Add unittests for querynode

### LINKED ISSUE #7337: Unittests code coverage to 60% in query node
#### Expected result:
Unittests code coverage to 60% in query node

#### Environment:
- Milvus version(e.g. v2.0.0-RC2 or 8b23a93):
- Deployment mode(standalone or cluster):
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:

### PR DESCRIPTION
**Pull request by bigsheeper**
Signed-off-by: bigsheeper <yihao.dai@zilliz.com>

Resolves: #7337

### CODE REVIEW

File: internal/querynode/collection.go [hunk 1]
```diff
@@ -79,21 +79,47 @@ func (c *Collection) removePartitionID(partitionID UniqueID) {
 }
 
 func (c *Collection) addVChannels(channels []Channel) {
-	log.Debug("add vChannels to collection",
-		zap.Any("channels", channels),
-		zap.Any("collectionID", c.ID()))
-	c.vChannels = append(c.vChannels, channels...)
+dstLoop:
```
Comment (czs007) [inline internal/querynode/collection.go:82]: better to name as OUTER:

File: internal/querynode/collection.go [hunk 2]
```diff
@@ -79,21 +79,47 @@ func (c *Collection) removePartitionID(partitionID UniqueID) {
...
 func (c *Collection) getVChannels() []Channel {
 	return c.vChannels
 }
 
 func (c *Collection) addPChannels(channels []Channel) {
-	log.Debug("add pChannels to collection",
-		zap.Any("channels", channels),
-		zap.Any("collectionID", c.ID()))
-	c.pChannels = append(c.pChannels, channels...)
+dstLoop:
```
Comment (czs007) [inline internal/querynode/collection.go:106]: better to name as OUTER:

File: internal/querynode/collection.go (hunk 1 above)
Comment (bigsheeper) [reply internal/querynode/collection.go]: okay

File: internal/querynode/collection.go (hunk 2 above)
Comment (bigsheeper) [reply internal/querynode/collection.go]: renamed

**Pull request merged**